plugins/scaffolder/package.json

```diff
@@ -73,6 +73,7 @@
     "@rjsf/validator-ajv8": "5.7.3",
     "@types/react": "^16.13.1 || ^17.0.0",
     "@uiw/react-codemirror": "^4.9.3",
+    "@zip.js/zip.js": "^2.7.24",
```

Comment:
It's fairly large this dep, I am wondering if there are better options. For example there's https://www.npmjs.com/package/jszip, but I don't know if that's necessarily better. It does seem like `CompressionStream` alone isn't gonna help us if we want to generate a zip file, although it does make me wonder if a gzip would be fine instead?

Reply:
jszip is only 100k and looks like a good replacement
https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js

Reply:
replaced with jszip